docs/gwcs/using_wcs.rst

```diff
@@ -74,6 +74,39 @@ Calling the :meth:`~gwcs.WCS.footprint` returns the footprint on the sky.
    ...    [  5.63010439, -72.05426843],
    ...    [  5.610708  , -72.04173847]])
 
+.. warning::
+
+  GWCS and astropy default to different tuple ordering conventions for representing
+  multi-dimensional bounding boxes. 
+  
+  * GWCS uses the ``"F"`` ordering convention, where the tuples are ordered
+    ``((x0min, x0max), (x1min, x1max), ..., (xnmin, xnmax))`` (x,y,z ordering).
+  * While astropy uses the ``"C"`` ordering convention, where tuples are ordered
```

Comment:
You mean specifically `astropy.modeling`, right? Things might be different in `astropy.wcs`. Should clarify.